src/state/slices/targetsSlice.js

```diff
@@ -14,6 +14,11 @@ const targetSlice = createSlice({
         targets: [..._state.targets, target],
       };
     });
+    builder.addMatcher(isAnyOf(getTargetsFulfilled), (_state, { payload: { targets } }) => {
+      return {
+        targets: targets.map(target => target.target),
```

Comment:
Maybe this could be written better as: 

`targets: targets.map(({ target }) => target),`